freqtrade/commands/strategy_utils_commands.py

```diff
@@ -39,4 +39,4 @@ def start_strategy_update(args: Dict[str, Any]) -> None:
     for filtered_strategy_obj in filtered_strategy_objs:
         # Initialize backtesting object
         instance_strategy_updater = StrategyUpdater()
-        StrategyUpdater.start(instance_strategy_updater, config, filtered_strategy_obj)
+        self.start(config, filtered_strategy_obj)
```

Comment:
well no - `instance_strategy_updater.start()` - self refers to ... "nothing" in this case (it's not a class)